src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -19,6 +20,8 @@ public DefaultServer(List<Type> typesToInclude)
             this.typesToInclude = typesToInclude;
         }
 
+        protected bool DisableDiagnostics = true;
```

Comment:
Do we actually over inherit from DefaultServer? I thought the publisher uses the default server but does not inherit from. Feels a bit strange that this assumes inheritance although we probably never actually do that in the wild.